cdap-ui/app/directives/old-log-viewer/log-viewer.html

```diff
@@ -0,0 +1,51 @@
+<!--
+  Copyright © 2015 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="row">
+  <div class="col-xs-2">
+
+    <ul class="nav nav-pills nav-stacked">
+      <li role="presentation"
+          ng-repeat="f in filters"
```

Comment:
Can this be more descriptive? I don't know what `filters` or `f` are representing.